test/examples/ucp_hello_world.c

```diff
@@ -235,6 +258,26 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
     printf("%s", (char *)(msg + 1));
     printf("\n\n---------------------------\n\n");
 
+    if (err_handling_opt.server_failure) {
```

Comment:
maybe better to add option to run the test in "ping-pong" mode, regardless of failure?
if we are not doing same data transfer in case of failure it's not a good example. need to show how failure can be handled w/o changing the data transfer logic.